.gitignore

```diff
@@ -226,3 +226,6 @@ CVS
 .ssh/*
 id_rsa*
 id_dsa*
+
+# Carthage
+Carthage/Build
```

Comment:
I was following the standard set by Alamofire. See https://github.com/Alamofire/Alamofire/blob/master/.gitignore#L39 Any reason why you want the trailing slash?